policyengine_us/parameters/gov/states/ca/tax/income/alternative_minimum_tax/amti_sources.yaml

```diff
@@ -0,0 +1,20 @@
+description: California adds the following items to calculate the total adjustments for the alternative minimum taxable income.
+values:
+   2021-01-01:
+     - medical_expense_deduction # line 2 Medical and dental expenses
+     - real_estate_taxes # line 3 Personal property taxes and real property taxes
+     - mortgage_interest # line 4 Certain interest on a home mortgage
+     - misc_deduction # line 5 Miscellaneous itemized deductions
+
+
+metadata:
+   unit: list  
+   period: year 
+   label: California altenative minimum taxable income adjustments 
+   references: 
+    - title: 2021 Schedule P (540) Alternative Minimum Tax and Credit Limitations - Residents
+      href: https://www.ftb.ca.gov/forms/2021/2021-540-p.pdf#page=1
+    - title: 2022 Schedule P (540) Alternative Minimum Tax and Credit Limitations - Residents
+      href: https://www.ftb.ca.gov/forms/2022/2022-540-p.pdf#page=1
+    - title: Cal. Rev. & Tax. Code § 17062 (c)
+      href: https://casetext.com/statute/california-codes/california-revenue-and-taxation-code/division-2-other-taxes/part-10-personal-income-tax/chapter-21-alternative-minimum-tax/section-17062-alternative-minimum-tax
```

Comment:
i don't see these mentioned here

Reply:
I am unsure on how to read the legal code here:
https://law.justia.com/codes/california/2022/code-rtc/division-2/part-10/chapter-2-1/section-17062/

ChatGPT wasnt too helpful, is this implying that the CA AMTI is a percentage of federal AMT (which would not match the amounts in the forms)